src/formBuilder/tributeInput.jsx

```diff
@@ -83,16 +83,23 @@ const TributeInput = props => {
 
     const getInitial = async () => {
       setLoading(true);
-      const tokenService = TokenService({
+
+      const tokenContract = createContract({
+        address: tributeToken,
+        abi: LOCAL_ABI.ERC_20,
         chainID: daochain,
-        tokenAddress: tributeToken,
-      });
-      const allowanceRes = await tokenService('allowance')({
-        accountAddr: address,
-        contractAddr: daoid,
       });
-      const decimalRes = await tokenService('decimals')();
-      const balanceRes = await tokenService('balanceOf')(address);
+
+      const allowanceRes = await tokenContract.methods
+        .allowance({
```

Comment:
these methods don't take an object, just args.

should be:

`.allowance(address, daoid).call()